mempool/src/shared_mempool/network.rs

```diff
@@ -507,7 +512,10 @@ impl<NetworkClient: NetworkClientInterface<MempoolSyncMsg>> MempoolNetworkInterf
                             std::cmp::Ordering::Greater
                         }
                     });
-
+                    info!(
+                        "Broadcasting buckets {:?} to peer {:?}",
+                        sender_buckets, peer
```

Comment:
is this for debug? typically we don't format with :?, it can be very verbose

Reply:
Removed it.